code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -128,6 +128,10 @@ def __init__(self, **kwargs) -> None:
             "AZURE_FORM_RECOGNIZER_KEY"
         )
         # Azure App Insights
+        self.APPINSIGHTS_ENABLED = (
```

Comment:
It could be nice to have this logic defined a single time, rather than manually process each value. I did [something similar](https://github.com/microsoft/rag-experiment-accelerator/blob/development/rag_experiment_accelerator/config/environment.py#L77) in rag-experiment-accelerator recently, if this seems helpful I could create a task for this refactoring

Reply:
Good point! Let me make that change.

Reply:
Didn't want to make your PR bigger than it needs to be, happy to create an issue for this instead if helpful

Reply:
I've made a small change to have a helper for env var booleans. But an issue for the rest would be good!

Reply:
https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/issues/515